manager/integration/tests/test_rwx.py

```diff
@@ -119,11 +626,82 @@ def test_rwx_statefulset_scale_down_up():
     11. Wait for new pods to come up.
     12. Check the data md5sum in new pods
     """
-    pass
+    nfs_provisioner_pod = k8sclient.CoreV1Api().list_pod_for_all_namespaces(
+        label_selector="app=longhorn-nfs-provisioner", watch=False)
+    nfs_provisioner_pod_name = nfs_provisioner_pod.items[0].metadata.name
+
+    statefulset_name = 'statefulset-rwx-multi-pods-test'
+    statefulset['metadata']['name'] = \
+        statefulset['spec']['selector']['matchLabels']['app'] = \
+        statefulset['spec']['serviceName'] = \
+        statefulset['spec']['template']['metadata']['labels']['app'] = \
+        statefulset_name
+    statefulset['spec']['volumeClaimTemplates'][0]['spec']['storageClassName']\
+        = NFS_STORAGECLASS_NAME
+
+    create_and_wait_statefulset(statefulset)
+
+    command = "ls /export | grep -i 'pvc' | wc -l"
+
+    dir_num = read_nfs_provisioner_data(
+        core_api, command, nfs_provisioner_pod_name, 'longhorn-system')
+
+    assert dir_num == '2'
+
+    md5sum_pod = []
+    for i in range(2):
+        test_pod_name = statefulset_name + '-' + str(i)
+        test_data = generate_random_data(VOLUME_RWTEST_SIZE)
+        write_pod_volume_data(core_api, test_pod_name, test_data)
+        md5sum_pod.append(test_data)
+
+    statefulset['spec']['replicas'] = replicas = 0
+    apps_api = get_apps_api_client()
+    apps_api.patch_namespaced_stateful_set(
+        name=statefulset_name,
+        namespace='default',
+        body={
+            'spec': {
+                'replicas': replicas
+            }
+        })
+    for i in range(DEFAULT_STATEFULSET_TIMEOUT):
+        s_set = apps_api.read_namespaced_stateful_set(
+            name=statefulset['metadata']['name'],
+            namespace='default')
+        if s_set.status.ready_replicas == replicas or \
+                (replicas == 0 and not s_set.status.ready_replicas):
+            break
+        time.sleep(DEFAULT_STATEFULSET_INTERVAL)
+
+    dir_num = read_nfs_provisioner_data(
+        core_api, command, nfs_provisioner_pod_name, 'longhorn-system')
+    assert dir_num == '2'
+
+    statefulset['spec']['replicas'] = replicas = 2
+    apps_api = get_apps_api_client()
+    apps_api.patch_namespaced_stateful_set(
+        name=statefulset_name,
+        namespace='default',
+        body={
+            'spec': {
+                'replicas': replicas
+            }
+        })
+    wait_statefulset(statefulset)
+
+    for i in range(2):
+        test_pod_name = statefulset_name + '-' + str(i)
+        command = 'cat /data/test'
+        pod_data = read_nfs_provisioner_data(
+            core_api, command, test_pod_name, 'default')
+
+        assert pod_data == md5sum_pod[i]
 
 
 @pytest.mark.skip(reason="TODO")
-def test_rwx_offline_node_longhorn_nfs_provisioner():
+def test_rwx_offline_node_longhorn_nfs_provisioner(
+    core_api, nfs_storage_class, statefulset, pod):  # NOQA
```

Comment:
all the other functions params are indented an additional step

Reply:
Fixed it.